string sensor_name
string status
float32 confidence
